cli/config.go

```diff
@@ -89,7 +89,7 @@ func (w *configGetCmd) SetFlags(f *flag.FlagSet) {}
 
 func (w *configGetCmd) Execute(ctx context.Context, f *flag.FlagSet, _ ...interface{}) subcommands.ExitStatus {
 	if len(f.Args()) < 1 {
-		log.Warnln(w.Usage())
+		log.Warnf(ctx, "%s", w.Usage())
```

Comment:
I have no preference either way, but any reason for not doing a
```suggestion
		log.Warnf(ctx, w.Usage())
```
?

Reply:
If the usage string contains a percentage sign, then it would be interpreted as a format code. Using `"%s"` ensures the string is interpreted literally.